templates/data/kafka.html

```diff
@@ -0,0 +1,312 @@
+      </div>
+      <hr>
+      <div class="p-section">
+        <a class="p-button--positive" aria-controls="data-kafka-modal" href="/contact-us">Contact us</a>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--25-75 is-split-on-medium">
+    <hr class="p-rule">
+    <div class="col">
+      <h2 class="p-text--small-caps">Made to give you<br class="u-hide--small"> peace of mind</h2>
+    </div>
+    <div class="col">
+      <ul class="p-list--divided">
+        <li class="p-list__item p-heading--2">10 years security maintenance </li>
+        <li class="p-list__item p-heading--2">Open source</li>
+        <li class="p-list__item p-heading--2">Simple per node subscription</li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row">
+    <hr class="p-rule">
+    <div class="col-start-large-4 col-6">
+      <div class="p-section--shallow">
+        <h2>Hyper-automated Kafka<sup>&reg;</sup> with enterprise-grade support</h2>
+      </div>
+      <p>Charmed Kafka is simply the best way to run Kafka, on the cloud or in your data centre. Runs directly on cloud infrastructure or on Kubernetes.</p>
+      <hr class="p-rule is-muted">
+      <div class="p-section--shallow">
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Automated security and configuration best practice</li>
+          <li class="p-list__item is-ticked">Built-in observability</li>
+          <li class="p-list__item is-ticked">Multi-cloud</li>
+          <li class="p-list__item is-ticked">Self-healing</li>
+        </ul>
+      </div>
+      <p><a class="p-button" href="/data/docs/kafka/iaas">Get started with Charmed Kafka</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--25-75 is-split-on-medium">
+    <hr class="p-rule">
+    <div class="col">
+      <h2 class="p-text--small-caps">Integrate it with</h2>
+      <div class="p-logo-section">
+        <div class="p-logo-section__items">
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/bda8c6c1-terraform.png",
+              alt="",
+              width="411",
+              height="256",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"},
+              ) | safe
+            }}
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="col">
+      <h2 class="p-text--small-caps">Run Apache Kafka<sup>&reg;</sup> on</h2>
+      <div class="p-logo-section">
+        <div class="p-logo-section__items">
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/36a0cd3b-microsoft-azure-2021.png",
+              alt="",
```

Comment:
I'll leave this to @juanruitina but he usually tells me to fill these up with the name of the logo. [W3C accessibility](https://www.w3.org/WAI/tutorials/images/decorative/) seems to be impartial, but you could argue it from different points of view. This can be a decorative thing or it may not be a decorative thing if you want to sell that brand.

Reply:
These are not decorative given it's informative for users that can read, and there's not an alternative in text. So yes, we should add alt text for all these logos. Thanks a lot for flagging it up, Carlos. @anthonydillon Please use this: HashiCorp Terraform, Microsoft Azure, OpenStack, Google Cloud, VMware, AWS